opencti-platform/opencti-graphql/src/modules/ingestion/ingestion-taxii-domain.ts

```diff
@@ -103,3 +114,18 @@ export const ingestionDelete = async (context: AuthContext, user: AuthUser, inge
   });
   return ingestionId;
 };
+
+export const ingestionTaxiiResetState = async (context: AuthContext, user: AuthUser, ingestionId: string) => {
+  await patchTaxiiIngestion(context, user, ingestionId, { current_state_cursor: undefined });
+  const ingestionUpdated = await findById(context, user, ingestionId);
```

Comment:
You can probably save a call to elastic by using the return value of patchTaxiiIngestion instead of calling additional findById